crates/fluvio-test-util/test_meta/environment.rs

```diff
@@ -122,8 +131,8 @@ pub struct EnvironmentSetup {
     pub keep_cluster: bool,
 
     /// topic name used
-    #[structopt(short("t"), long, default_value = "topic")]
```

Comment:
why is removed?.

Reply:
It wasn't completely removed. I moved this default value to `crates/fluvio-test-util/test_runner/test_meta.rs` for #1544 since I needed this fixed to test.

I changed this to Option<String> so I could support a hierarchy for overriding the topic name the test should use and I needed the possibility for the value to be `None`.

1. If we provide `--topic-name`, then the test will use that value, overriding the `fluvio_test` macro
2. Next, if the `fluvio_test` macro providers a topic name, then use that value
3. Otherwise we'll default to the topic name `test`

Reply:
seems like this could be clean up. but it is not show stopper. let's merge first